website/docs/legacy/v4/guides/menu.md

```diff
@@ -35,7 +34,7 @@ To use it just call this helper in your template.
 ```js
 use Eightshift_Libs\Menu\Menu;
 
-Menu::bem_menu( 'header_main_default', 'main-navigation' );
+Menu::bemMenu( 'header_main_default', 'main-navigation' );
```

Comment:
```suggestion
Menu::bemMenu( 'header_main_nav', 'main-navigation' );
```
Since key was changed to `header_main_nav` in method example, perhaps it wouldn't be a bad idea to use the same key in this one 🙂

Reply:
Hey @infinum/wordpress-team, do you agree with this suggestion? If yes, I can commit it and approve this PR.

Reply:
agreed